sphinx/builders/latex/__init__.py

```diff
@@ -224,6 +224,8 @@ def init_multilingual(self) -> None:
                 self.context['substitutefont'] = '\\usepackage{substitutefont}'
             else:
                 self.context['textgreek'] = ''
+            if self.context['substitutefont'] == '':
+                self.context['fontsubstitution'] = ''
```

Comment:
fontsubstitution contains code using `\substitutefont` command from `substitutefont` package; if the latter is not loaded there is no need to insert the fontsubstitution code although it would be ok because it is condiional code, but not inserting something not useful is better